installer_scripts/install-yb-voyager

```diff
@@ -165,6 +167,35 @@ macos_main() {
 # COMMON
 #=============================================================================
 
+check_binutils_version() {
+	output "Checking binutils version."
+	min_required_version='2.25'
+	os=$1
+	
+	case ${os} in 
+		"centos")
+			# Example output of "ld -v" on CentOS/RHEL:
+			# GNU ld version 2.30-113.el8
+			version=$(ld -v | grep -o 'version [0-9.]\+' | awk -F' ' '{print $2}')
+			;;
+		"ubuntu")
+			# Example output of "ld -v" on Ubuntu:
+			# GNU ld (GNU Binutils for Ubuntu) 2.38
+			version=$(ld -v | grep -o '[0-9.]\+')
+			;;
+		*)
+			echo "ERROR: unsupported linux distro"
+			exit 1
+			;;
+	esac
+
```

Comment:
With the following approach, we can get rid of this case statement as well as distinction between centos/ubuntu:
```
[amit.jambure@dev-server-ajambure ~]$ ld -v | awk '{print $NF}'
2.27-44.base.el7_9.1
[amit.jambure@dev-server-ajambure ~]$ ld -v | awk '{print $NF}' | awk -F '-' '{print $1}'
2.27
[amit.jambure@dev-server-ajambure ~]$ echo -n "GNU ld (GNU Binutils for Ubuntu) 2.38" | awk '{print $NF}' | awk -F '-' '{print $1}'
2.38
```
`NF` is an awk built in variable which means number of "fields" in the current row. So `print $NF` outputs last word of a line. We then pass the last word again to awk--this time with `-` as a field separator and then take the first "field".

Reply:
Got that!

Reply:
Out of curiosity, found even shorter method:
```
[centos@ip-10-9-12-152 ~]$ ld -v | awk '{split($NF, a, "-"); print a[1]}'
2.27
[centos@ip-10-9-12-152 ~]$ echo -n "GNU ld (GNU Binutils for Ubuntu) 2.38" | awk '{split($NF, a, "-"); print a[1]}'
2.38
```
No need to make this change, though.